src/commands/infra/scaffold.ts

```diff
@@ -96,89 +100,49 @@ export const execute = async (
  *
  * @param command Commander command object to decorate
  */
-export const scaffoldCommandDecorator = (command: commander.Command): void => {
-  command
-    .command("scaffold")
-    .alias("s")
-    .description("Create initial scaffolding for cluster deployment.")
-    .option("-n, --name <name>", "Cluster name for scaffolding")
-    .option(
-      "-s, --source <cluster definition github repo>",
-      "Source URL for the repository containing the terraform deployment"
-    )
-    .option(
-      "-v, --version <repository version>",
-      "Version or tag for the repository so a fixed version is referenced"
-    )
-    .option(
-      "-t, --template <path to variables.tf> ",
-      "Location of the variables.tf for the terraform deployment"
-    )
-    .action(async (opts: ICommandOptions) => {
-      const config = Config();
-      await execute(config, opts, process.exit);
```

Comment:
Is there a reason why we abstract the command decorator in a `json` file?  Is this a best practice? It just seems like this is just another way of describing the scaffold command, but not necessarily a _better_ way. 

My only concern with doing this is, now we are introducing inconsistencies across `spk`. Do we plan on making this change for `spk infra generate`? or even other `spk` commands as well?

Reply:
the plan to move declaration like this out of code. I already did for generate.ts and will do for the rest of the command files as part of https://github.com/microsoft/bedrock/issues/885 effort

Reply:
Okay, thanks for the clarification!

Reply:
Thanks Yvonne, the epic is not clear until Andre pointed out to me that I have added more information. Sorry for the confusion.